tests/test_components/test_autograd.py

```diff
@@ -76,19 +77,29 @@
 DA_SHAPE = (DA_SHAPE_X, 1_000, 1_000) if TEST_CUSTOM_MEDIUM_SPEED else (DA_SHAPE_X, 12, 12)
 
 # number of vertices in the polyslab
-NUM_VERTICES = 100_000 if TEST_POLYSLAB_SPEED else 12
+NUM_VERTICES = 100_000 if TEST_POLYSLAB_SPEED else 30
 
 # sim that we add traced structures and monitors to
 SIM_BASE = td.Simulation(
-    size=(LX, 3, LZ),
-    run_time=1e-12,
+    size=(LX, 3.15, LZ),
+    run_time=200 / FWIDTH,
     sources=[
-        td.PointDipole(
+        # td.PointDipole(
+        #     center=(0, 0, -LZ / 2 + WVL),
+        #     polarization="Ey",
+        #     source_time=td.GaussianPulse(
+        #         freq0=FREQ0,
+        #         fwidth=FWIDTH,
+        #         amplitude=1.0,
+        #     ),
+        # ),
```

Comment:
Should this be removed or why is it commented out?

Reply:
Plane wave worked better for many numerical tests, and the point dipole created some havoc in the diffraction monitor. so for now I just commented it out and replaced it with the PlaneWave, but In the future I might want to add it back. I'll create a point dipole variable at the module level and remove this comment.